project/history.go

```diff
@@ -30,7 +30,11 @@ func FullVersion() string {
 var History relic.ImmutableHistory = relic.NewHistory("Hyperledger Burrow", "https://github.com/hyperledger/burrow").
 	MustDeclareReleases(
 		"",
-		`Release our mempool signing lock once transactions have been CheckTx'd' to massively increase throughput, also support mempool signing for BroadcastTxAsync.`,
+		`Release our mempool signing lock once transactions have been CheckTx'd' to massively increase throughput, also support mempool signing for BroadcastTxAsync.
+
+#### Breaking changes
```

Comment:
I know we used breaking changes in the past, but I thought we could just use:

\### Changed

As per: https://keepachangelog.com/en/1.0.0/